cal_test.go

```diff
@@ -6,6 +6,8 @@ import (
 	"reflect"
 	"testing"
 	"time"
+
+	"github.com/stretchr/testify/assert"
```

Comment:
Please remove this dependency. Currently there are no external dependencies for the library and I'd like to keep it that way.

Reply:
Done